src/__test__/integration/landing-page.test.ts

```diff
@@ -29,4 +30,20 @@ describe('landing page', () => {
     accordionItems[1].click()
     expect(window.getComputedStyle(accordionItems[0].querySelector('.panel')!).display).to.equal('none')
   })
+
+  // it('sends a request and displays the results when subscribing', () => {
+
+  //   fetchMock.mock({ url: '/v1/subscribe', method: 'POST' }, 'OK')
```

Comment:
I think I see what you were chatting about. For the extension, the tests manually call the background/run and popup/mount functions which are running in as part of the same javascript context and thus have access to the same global object. The setup here is a bit different where JSDOM is fetching the resources and running them itself. 

So I think you can `stub(window, 'fetch')` to make it do what you want for this purpose